docs/source/guides/statistical_insights.ipynb

```diff
@@ -209,6 +209,209 @@
    "source": [
     "With the normally distributed set of data, the likelyhood of outliers is closer to what we'd expect, around .7%. "
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inferring Frequency from Noisy Timeseries Data\n",
+    "\n",
+    "`df.ww.infer_temporal_frequencies` will infer the observation frequency (daily, biweekly, yearly, etc) of each temporal column, even on noisy data. If a temporal column is predominantly of a single frequency, but is noisy in any way (ie. contains duplicate timestamps, nans, gaps, or timestamps that do not align with overall frequency), this table accessor will provide the most likely frequency, and the rows of data that do not adhere to this frequency."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Inferring Non-Noisy data\n",
+    "If your timeseries data is perfect and doesn't contain any noisy data, `df.ww.infer_temporal_frequencies()` will return a dictionary where the keys are the columns of each temporal column and the value is the pandas alias string."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.DataFrame({\n",
+    "    \"idx\": range(100),\n",
+    "    \"dt1\": pd.date_range(\"2005-01-01\", periods=100, freq=\"H\"),\n",
+    "    \"dt2\": pd.date_range(\"2005-01-01\", periods=100, freq=\"B\")\n",
+    "})\n",
+    "df.ww.init()\n",
+    "df.ww.infer_temporal_frequencies()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "{\n",
+    "    'dt1': 'H', \n",
+    "    'dt2': 'B'\n",
+    "}\n",
```

Comment:
The code cells will get executed when the docs are built, and the output here will be shown under the code cell. It will have slightly less nice formatting, but is still pretty readable to me. 

![image](https://user-images.githubusercontent.com/64278226/158890100-a7897740-5b98-4c7c-b71c-c877b089c503.png)

Reply:
removed.